frame/contracts/src/exec.rs

```diff
@@ -1571,7 +1572,8 @@ mod tests {
 			);
 
 			// Check that the account has not been created.
-			assert!(Storage::<Test>::code_hash(&instantiated_contract_address).is_err());
+			assert_noop!(Storage::<Test>::code_hash(&instantiated_contract_address),
+				ContractAbsentError);
```

Comment:
```suggestion
			assert_noop!(
				Storage::<Test>::code_hash(&instantiated_contract_address),
				ContractAbsentError,
			);
```

or put it on a single line